buf.yaml

```
@@ -0,0 +1,6 @@
+version: v2
+modules:
+  - path: proto
+    name: buf.build/bufbuild/modules-proto
```

This is another option

```suggestion
    name: buf.build/bufbuild/managed-modules
```

---

CI seems to have created a `modules-proto` module, we'll want to delete that.

---

Correct, it's the one I'm using in the backend for testing in the other PR. I'll delete it when I can replace it with this one.

---

Renamed